translations/messages.fr.xlf

```diff
@@ -497,93 +497,86 @@
+                <target>Gagnez du temps sur la rédaction de vos arrêtés et communiquez plus efficacement avec les usagers de la route.</target>
             </trans-unit>
             <trans-unit id="landing.how-it-works.city.link">
                 <source>landing.how-it-works.city.link</source>
                 <target>Créer un compte pour ma commune</target>
             </trans-unit>
             <trans-unit id="landing.how-it-works.gps.title">
                 <source>landing.how-it-works.gps.title</source>
-                <target>Pour les GPS</target>
+                <target>Pour les services numériques</target>
             </trans-unit>
             <trans-unit id="landing.how-it-works.gps.description">
                 <source>landing.how-it-works.gps.description</source>
-                <target><![CDATA[<b>]]>L’intégration<![CDATA[</b>]]> des réglementations dans les <![CDATA[<b>]]>calculateurs d'itinéraires<![CDATA[</b>]]> favorise la bonne circulation.</target>
+                <target>Accédez à la réglementation de circulation et de stationnement des territoires partenaires, dans des formats adaptés à vos besoins.</target>
             </trans-unit>
             <trans-unit id="landing.how-it-works.gps.link">
                 <source>landing.how-it-works.gps.link</source>
                 <target>Accéder aux données Open Data</target>
             </trans-unit>
-            <trans-unit id="landing.how-it-works.trucks.title">
-                <source>landing.how-it-works.trucks.title</source>
-                <target>Pour les poids lourds</target>
+            <trans-unit id="landing.road-users.title">
+                <source>landing.road-users.title</source>
+                <target>Pour les usagers de la route</target>
             </trans-unit>
-            <trans-unit id="landing.how-it-works.trucks.description">
-                <source>landing.how-it-works.trucks.description</source>
-                <target>Les chauffeurs ont enfin un <![CDATA[<b>]]>moyen fiable de connaître la réglementation<![CDATA[</b>]]>, directement via leurs GPS.</target>
+            <trans-unit id="landing.road-users.description">
+                <source>landing.road-users.description</source>
+                <target>Soyez mieux informés des conditions de circulation.</target>
             </trans-unit>
-            <trans-unit id="landing.how-it-works.trucks.link">
-                <source>landing.how-it-works.trucks.link</source>
-                <target>Voir les GPS partenaires</target>
+            <trans-unit id="landing.road-users.link">
+                <source>landing.road-users.link</source>
+                <target>Comprendre nos missions</target>
             </trans-unit>
             <trans-unit id="landing.itinerary.title">
                 <source>landing.itinerary.title</source>
-                <target>Des itinéraires adaptés pour une circulation apaisée</target>
+                <target>Comment ça marche ?</target>
```

Comment:
Certaines des traductions changées ici font que la clé et la traduction ne correspondent plus entre elles

Par exemple ici "Comment ça marche ?" devrait être raccordé à la clé "landing.how-it-works.title" qui est désormais traduit par "Pour qui ?" (alors que cette traduction devrait être liée à "landing.for-who.title" par exemple)

Par contre les traductions de la partie landing.road-users.* correspondent bien

Il faudrait mettre en cohérence le reste des traductions aussi